### PR TITLE
Feature/21520 update dev file manifest

### PR DESCRIPTION
**Pull request by timwright12**
## Description
Update dev bucket URL

### CODE REVIEW

**Review by alexandec — APPROVED** [review]

File: src/site/constants/buckets.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const { VAGOVDEV, VAGOVSTAGING, VAGOVPROD } = require('./environments');
 
 module.exports = {
-  [VAGOVDEV]: 'https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com',
+  [VAGOVDEV]: 'https://s3-us-gov-west-1.amazonaws.com/apps.dev.va.gov',
```
Comment (U-DON) [inline src/site/constants/buckets.js:4]:
```suggestion
  [VAGOVDEV]: 'https://apps.dev.va.gov.s3-us-gov-west-1.amazonaws.com',
```

Comment (timwright12) [reply src/site/constants/buckets.js:4]: @U-DON that format doesn't actually work: https://apps.dev.va.gov.s3-us-gov-west-1.amazonaws.com/generated/file-manifest.json

Comment (U-DON) [reply src/site/constants/buckets.js:4]: I'm able to access it, but it does have a cert issue. Didn't realize putting the bucket name in the path was a thing.

**Review by U-DON — APPROVED** [review]

File: src/site/constants/buckets.js (hunk 1 above)
Comment (timwright12) [reply src/site/constants/buckets.js:4]: Yeah it's kind of a weird thing, but this is the discussion with ops https://dsva.slack.com/archives/CBU0KDSB1/p1615926861056600?thread_ts=1615827961.141400&cid=CBU0KDSB1

**Pull request merged**